modules/ethereumNode.js

```diff
@@ -527,7 +517,7 @@ class EthereumNode extends EventEmitter {
         try {
             return fs.readFileSync(fullPath, {encoding: 'utf8'});
         } catch (err){
-            log.error(`Unable to read from ${fullPath}`, err);
+            log.error(`Unable to read from ${fullPath}`);
```

Comment:
This error will always no appear for installations where it can't load the `daoFork` file. To make this error less huge in the console i removed the `err` object, especially if the loading file error should be pretty clear why it happens.

Otherwise we could build in a check, if `daoFork` don't log the error